packages/framework/src/Component/Translation/JsFileExtractor.php

```diff
@@ -6,63 +6,48 @@
 use JMS\TranslationBundle\Model\Message;
 use JMS\TranslationBundle\Model\MessageCatalogue;
 use JMS\TranslationBundle\Translation\Extractor\FileVisitorInterface;
-use PLUG\JavaScript\JParser;
-use PLUG\JavaScript\JTokenizer;
-use Shopsys\FrameworkBundle\Component\Javascript\Parser\Translator\JsTranslatorCallParser;
 use SplFileInfo;
 use Twig_Node;
 
 class JsFileExtractor implements FileVisitorInterface
 {
-    /**
-     * @var \SplFileInfo
-     */
-    protected $file;
+    public const DUMP_FILE = 'translationsDump.json';
+
+    public const DEFAULT_MESSAGE_DOMAIN = 'messages';
 
     /**
      * @var \JMS\TranslationBundle\Model\MessageCatalogue
      */
     protected $catalogue;
 
-    /**
-     * @var \Shopsys\FrameworkBundle\Component\Javascript\Parser\Translator\JsTranslatorCallParser
-     */
-    protected $jsTranslatorCallParser;
-
-    /**
-     * @param \Shopsys\FrameworkBundle\Component\Javascript\Parser\Translator\JsTranslatorCallParser $jsTranslatorCallParser
-     */
-    public function __construct(JsTranslatorCallParser $jsTranslatorCallParser)
-    {
-        $this->jsTranslatorCallParser = $jsTranslatorCallParser;
-    }
-
     /**
      * @param \SplFileInfo $file
      * @param \JMS\TranslationBundle\Model\MessageCatalogue $catalogue
      */
     public function visitFile(SplFileInfo $file, MessageCatalogue $catalogue)
     {
-        if (substr($file, -3) !== '.js') {
+        if (substr($file, -strlen(static::DUMP_FILE)) !== static::DUMP_FILE) {
```

Comment:
This condition is really hard to understand to me and I'm not really sure what this checking.
Can you please extract it into a separate method and name accordingly?

Reply:
c192df736b379e76439b97301e3593197d8691c7